### ISSUE TITLE
signing verification fails with test vectors

### ISSUE THREAD

**Issue by sacketty** [kjur/jsrsasign#350]
using test vectors from [https://www.ietf.org/rfc/rfc6979.txt](url) and considering the NIST P-384 curve with 

> With SHA-512, message = "sample"

we extract the signature with : 
>var testSignature = r.ECDSA.hexRSSigToASN1Sig(rVal, sVal)

as result
>testSignature="MGUCMQDtCVnViAqy2Gmuf2wpFcbWD5ZQf5yz4EfABGhh2kp5nP4w81zJAAVtfJnNeIJDNwkCMFEsjM7uOJCoQFjOHiLbwhmPQjI86KypE1Mp8DwGjlES3HzD7zRG3vzrAaRcJmf91Q=="

and the hex public key with the formulae:
>var testPubKey = "04" + Ux + Uy;

which gives
>testPubKey="04EC3A4E415B4E19A4568618029F427FA5DA9A8BC4AE92E02E06AAE5286B300C64DEF8F0EA9055866064A254515480BC138015D9B72D7D57244EA8EF9AC0C621896708A59367F9DFB9F54CA84B3F1C9DB1288B231C3AE0D4FE7344FD2533264720"

```
var rVal = "ED0959D5880AB2D869AE7F6C2915C6D60F96507F9CB3E047C0046861DA4A799CFE30F35CC900056D7C99CD7882433709"
var sVal = "512C8CCEEE3890A84058CE1E22DBC2198F42323CE8ACA9135329F03C068E5112DC7CC3EF3446DEFCEB01A45C2667FDD5"
var Ux = "EC3A4E415B4E19A4568618029F427FA5DA9A8BC4AE92E02E06AAE5286B300C64DEF8F0EA9055866064A254515480BC13"
var Uy = "8015D9B72D7D57244EA8EF9AC0C621896708A59367F9DFB9F54CA84B3F1C9DB1288B231C3AE0D4FE7344FD2533264720"
var testSignature = r.ECDSA.hexRSSigToASN1Sig(rVal, sVal)
var testPubKey = "04" + Ux + Uy;
var signature = new r.Signature({alg: "SHA512withECDSA"});
signature.init({xy: testPubKey, curve: "secp384r1"});
var message = "sample";
signature.updateString(message);
var result = signature.verify(testSignature);
console.log("result:", result)
```
it returns 
> $ false.

I have checked the same test vector with other libraries (Ruby, Java) . And it works
Any clue ?

**Comment by JerryUS2019** [comment]
@sacketty Is this still an issue?

**Comment by kjur** [comment]
This issue was fixed in 8.0.15 release. Thank you.
